RocketChat/Rocket.Chat
# [FIX] Remove doubled "Canned Responses" strings

## Proposed changes (including videos or screenshots)


 - Remove doubled canned response setting introduced in #22703 (by setting id change);
 - Update "Canned Responses" keys to "Canned_Responses".



## Issue(s)

[Task - ClickUp](https://app.clickup.com/t/7084cm)

## Steps to test or reproduce

Go to Administration and search for 'Canned Responses'. Only **one** setting (menu) should be displayed in the sidebar.

## Further comments
